CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+1.0.0 (March 2018)
+------------------
+
+This version includes:
+
+
+- displayed pixels intensity when mousing over image
+- enabled viewing of multiple images in the same browser window by double-clicking
+- added support for opening the image in other installed applications e.g. OMERO.figure
+- added option to export ROIs statistics in CSV file or Excel file
+- improved ROIs creation and manipulation and persistence
+- added new icon for polyline
+- improved ROIs statistics export
```

Comment:
I'm not sure we need to have "Added ROIs statistics" and "Improved ROIs statistics" in the same list?

Reply:
That one is more for the handling of the values.
It needs to be rephrased